openverse_catalog/dags/common/loader/sql.py

```diff
@@ -151,12 +152,15 @@ def load_s3_data_to_intermediate_table(
               'us-east-1'
             );
             """
-        )
+        ),
+        handler=lambda c: c.fetchone()[0],
```

Comment:
Cool!

Reply:
It's kind of annoying that this isn't something immediately available (I could have sworn that `DbApiHook` did this by default before...) but it's nice that we can add it in where necessary!